Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -61,6 +61,10 @@ theorem star_comm_self' [Mul R] [Star R] (x : R) [IsStarNormal x] : star x * x =
   IsStarNormal.star_comm_self
 #align star_comm_self' star_comm_self'
 
+lemma isStarNormal_iff [Mul R] [Star R] {x : R} :
+    IsStarNormal x ↔ star x * x = x * star x :=
```

Comment:
I'd be tempted to keep `Commute (star x) x` on the RHS, to match the definition. You could even use `@[mk_iff]` to generate this lemma automatically.